shell/platform/fuchsia/dart-pkg/zircon/lib/src/system.dart

```diff
@@ -86,7 +86,8 @@ class ReadResult extends _Result {
   /// Returns the bytes as a Uint8List. If status != OK this will throw
   /// an exception.
   Uint8List bytesAsUint8List() {
-    return _bytes!.buffer.asUint8List(_bytes!.offsetInBytes, _numBytes!);
+    final lbytes = bytes;
+    return lbytes.buffer.asUint8List(lbytes.offsetInBytes, _numBytes!);
```

Comment:
Remove `lbytes` and replace it with `bytes`

Reply:
wouldn't that be a little less efficient with two getter calls involving null checks?

Reply:
Either is fine - not sure if it matters for performance

Reply:
Added @stereotype441 for his take on which form would be better.

Reply:
I don't have a lot to add about which form would be better. @a-siva is right that two getter calls involving null checks might in principle be slightly worse in performance. Then again, it's possible that due to inlining, the two null checks will be recognized by an optimization pass as redundant, in which case it might not matter at all. It's also possible that this is simply not a hot enough code path to have a noticeable effect.

So lgtm either way.